apps/admin/controller.js

```diff
@@ -31,6 +32,53 @@ app.get('/users', (req, res, next) => {
   });
 });
 
+app.get('/email', (req, res) => {
+  const error = req.session.message;
+
+  delete req.session.message;
+  res.render('admin/email.html', { req });
+});
+
+app.post('/email', (req, res, next) => {
+  const query = { 'owner.userId': req.session.auth.userId };
+  const template = req.body.content;
+  const context = {
+    site: {
+      url: `${req.protocol}://${req.get('host')}`,
+    }
+  };
+
+  if (req.body.send) {
+    // @TODO
+  } else {
+    ApiUser.findOne(query, (err, api) => {
+      if (err) { return next(err); }
+
+      context.api = api;
+      context.user = api.contact;
+
+      sendgrid.renderTemplate(template, context, (err, data) => {
+        if (err && err.name == 'Template render error') {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Turbasen/Developer/file/2497055906/issues/source?bid=3263028&fileBranchId=3268038#l61)